app/app/src/main/res/layout/activity_main.xml

```diff
@@ -6,70 +6,37 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".ui.MainActivity">
-    <TextView
-        android:id="@+id/title_text_view"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/login_title_text_margin"
-        android:text="@string/title"
-        android:textAlignment="center"
-        android:textColor="@color/black"
-        android:textSize="@dimen/login_title_text_size"
-        android:textStyle="bold"
-        app:layout_constraintEnd_toEndOf="parent"
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/details_fragment_tool_bar"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-    <EditText
-        android:id="@+id/username_edit_text"
-        android:layout_width="@dimen/login_input_width"
-        android:layout_height="@dimen/login_input_height"
-        android:layout_marginTop="@dimen/login_input_margin_top"
-        android:background="@drawable/text_edit_border"
-        android:textColor="@color/general_tint"
-        android:textColorHint="@color/general_tint"
-        android:paddingLeft="@dimen/login_input_padding"
-        android:paddingRight="@dimen/login_input_padding"
-        android:hint="@string/enter_username_text"
-        android:textSize="@dimen/login_input_text_size"
-        android:inputType="text"
-        android:autofillHints=""
-        android:importantForAutofill="no"
+        app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="@dimen/login_constraint_horizontal_bias"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/title_text_view" />
-    <EditText
-        android:id="@+id/password_edit_text"
-        android:layout_width="@dimen/login_input_width"
-        android:layout_height="@dimen/login_input_height"
-        android:layout_marginTop="@dimen/login_input_password_margin_top"
-        android:background="@drawable/text_edit_border"
-        android:hint="@string/enter_password_text"
-        android:paddingLeft="@dimen/login_input_padding"
-        android:paddingRight="@dimen/login_input_padding"
-        android:textColor="@color/general_tint"
-        android:textColorHint="@color/general_tint"
-        android:textSize="@dimen/login_input_text_size"
-        android:inputType="textPassword"
-        android:autofillHints=""
-        android:importantForAutofill="no"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <androidx.appcompat.widget.Toolbar
+            app:logo="@drawable/ic_back_arrow"
```

Comment:
not correct. Your Toolbar should have an ImageView and a TextView. The ImageView should have the ic_back_arrow inside. Think of adding a ConstraintLayout inside the Toolbar and then inside the ConstraintLayout add the ImageView and the TextView.